### PR TITLE
Update .editorconfig

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by keradus** [comment]
:+1:

File: .editorconfig [hunk 1]
```diff
@@ -5,3 +5,9 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+
+[/Symfony/CS/Tests/Fixtures/**/*]
+end_of_line = ignore
+insert_final_newline = ignore
+indent_style = ignore
+trim_trailing_whitespace = ignore
```
Comment (keradus) [inline .editorconfig]: does `ignore` config value a valid one?
I cannot find it https://github.com/editorconfig/editorconfig/search?utf8=✓&q=ignore

Comment (localheinz) [inline .editorconfig]: Maybe this helps: http://stackoverflow.com/a/30539967/1172545?

Comment (localheinz) [inline .editorconfig]: What I meant is to create an `.editorconfig` with the following content

```yml
root = true
```

in [`/Symfony/CS/Tests/Fixtures`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/1.13/Symfony/CS/Tests/Fixtures) - maybe that helps?

Comment (SpacePossum) [inline .editorconfig]: >does ignore config value a valid one?

Comment (keradus) [inline .editorconfig]: so, would "official way" works for us ?

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**